packages/ui-react-rx/src/Api/index.tsx

```diff
@@ -25,10 +25,16 @@ type State = ApiProps & {
   subscriptions: Array<any> // rxjs$ISubscription | null>;
 };
 
+export function shouldUseLatestChain(chain?: string): boolean {
+  if (typeof chain === 'undefined') return false;
+  const re = new RegExp("(poc-1)", "i");
+  const match = re.test(chain.toLowerCase());
+  if (match) return false;
+  return true;
+}
```

Comment:
As per above, give this function some breathing room, it is very condensed for no real benefit.